src/components/Reports/Reports.js

```diff
@@ -27,22 +27,24 @@ const Reports = ({ dependencies: { datahubClient } }) => {
   const [state, setState] = useState({
     periodSelectedDays: periodSelectedDaysDefault,
     dateFrom: addDays(today, periodSelectedDaysDefault * -1),
-    dateTo: today,
+    dateTo: periodSelectedDaysDefault ? today : new Date(),
+    isTodaySelected: !periodSelectedDaysDefault,
   });
 
   const changeDomain = async (name) => {
     const domainFound = state.domains.find((item) => item.name === name);
     setState((prevState) => ({ ...prevState, domainSelected: domainFound }));
   };
 
-  const changePeriod = (days) => {
+  const changePeriod = (daysToFilter) => {
```

Comment:
I think that in this case, `daysToFilter` is too much overloaded. 

What do you think about receiving `dateFrom` and `dateTo` parameters?

```suggestion
  const changePeriod = (dateFrom, dateTo, isDailyView) => {
```

Reply:
Estuvimos hablando y por la estructura del código no tiene sentido hacer eso. Esta función es la encargada de convertir la selección del combo en los valores correspondientes.